src/utils/jsHelper/helper.tsx

```
@@ -612,3 +612,31 @@ export function getExpiryDate(
+
+	const label = proofs
+		.map((p) =>
+			p
+				.replace(/([A-Z])/g, ' $1')
+				.replace(/^./, (str) => str.toUpperCase())
+		)
+		.join(' / ');
+	const requiredMark = isRequired ? ' *' : '';
+	return `Document for ${codes.join(', ')} (${label} )${requiredMark}`;
+};
+export const parseDocList = (list: any[], fromEligibility = false) => {
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)